_includes/pbc/all/install-features/202311.0/install-the-warehouse-picking-product-feature.md

```diff
@@ -20,9 +20,10 @@ Install the required features:
 
 Enable the following plugins.
 
-| PLUGIN                                                              | SPECIFICATION                                                                           | PREREQUISITES | NAMESPACE                                                                                      |
-|---------------------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------|------------------------------------------------------------------------------------------------|
-| ConcreteProductsByPickingListItemsBackendResourceRelationshipPlugin | Adds `concrete-products` resources as a relationship to `picking-list-items` resources. |               | Spryker\Glue\ProductsBackendApi\Plugin\GlueBackendApiApplicationGlueJsonApiConventionConnector |
+| PLUGIN                                                              | SPECIFICATION                                                                                     | PREREQUISITES | NAMESPACE                                                                                                                       |
+|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------|
+| ConcreteProductsByPickingListItemsBackendResourceRelationshipPlugin | Adds `concrete-products` resources as a relationship to `picking-list-items` resources.           |               | Spryker\Glue\ProductsBackendApi\Plugin\GlueBackendApiApplicationGlueJsonApiConventionConnector                                  |
+| ConcreteProductImageSetsByProductsBackendResourceRelationshipPlugin | Adds `concrete-product-image-sets` resources as a relationship to `picking-list-items` resources. |               | Spryker\Glue\ProductsProductImageSetsBackendResourceRelationship\Plugin\GlueBackendApiApplicationGlueJsonApiConventionConnector |
```

Comment:
* This plugin would be a part of another module (ProductImageSetsBackendApi), the folder structure would be the same. 
* We have to change the namespaces in the code blocks.

Reply:
What do you mean? Right now it is in ProductsProductImageSetsBackendResourceRelationship module
Should we make it ProductImageSetsBackendApi now for the future changes?

Reply:
Let's put it on hold then, till the plugin not moved to the proper module.